src/constants.py

```diff
@@ -18,3 +19,7 @@
 
 WORKLOAD_CONTAINER = "glauth"
 WORKLOAD_SERVICE = "glauth"
+
+DEFAULT_UID = 5001
+DEFAULT_GID = 5501
```

Comment:
I chose these numbers (from the glauth example) as the default uid and gid for the bind account creation. Linux generally reserves a range for systems. The left is for users to allocate.

See:
- https://en.wikipedia.org/wiki/User_identifier
- https://en.wikipedia.org/wiki/Group_identifier